apps/mull-ui/src/app/pages/login/login.scss

```diff
@@ -0,0 +1,142 @@
+@import 'variables';
+
+.login-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 1.5rem;
+  padding-top: 2.5rem;
+  padding-left: 2rem;
+  padding-right: 2rem;
+}
+
+.facebook {
+  margin-bottom: 2rem;
+}
+
+.custom-text-input-container,
+.facebook,
+.google,
+.login,
+.separator,
+.twitter {
+  align-self: center;
+  justify-self: center;
+}
+
+.separator {
+  display: flex;
+  align-items: center;
+  color: $primary-color;
+  width: 100%;
+}
+
+.separator::before,
+.separator::after {
+  content: '';
+  flex: 1;
+  border-bottom: 1px solid $primary-color;
+}
+
+.separator::before {
+  margin-right: 0.25rem;
+}
+.separator::after {
+  margin-left: 0.25rem;
+}
+
+.login,
+.twitter,
+.google,
+.facebook {
+  width: 100%;
+  height: 45px;
+  cursor: pointer;
+  border-radius: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+}
+
+.twitter,
+.google,
+.facebook {
+  color: white;
+  font-weight: bold;
+}
+
+.login {
+  font-weight: bold;
+  color: black;
+  background-color: rgba($color: $primary-color, $alpha: 0.6);
+  outline: none;
+}
+
+.twitter {
+  background-color: $twitter;
+}
+
+.google {
+  background-color: $google;
+}
+
+.facebook {
+  background-color: $facebook;
+  padding-left: 1.2rem;
+}
+
+.login-mull-logo {
+  padding-top: 2rem;
+  display: block;
+  margin: 0 auto;
+  width: 30%;
+  padding-bottom: 3rem;
+}
+
+.sign-up {
+  text-align: center;
+  font-size: 14px;
+}
+
+.register-login-page {
+  font-size: 14px;
+  text-decoration: none;
+  color: $primary-color;
+  font-weight: bold;
+}
+
+@media (min-width: 600px) {
+  .login-container {
+    padding-top: 0;
+  }
+  .login-mull-logo {
+    padding-top: 5rem;
+  }
+  .login,
+  .twitter,
+  .google,
+  .facebook,
+  .separator,
+  .custom-text-input-container {
+    width: 45%;
+  }
+
+  .login-mull-logo {
+    width: 12%;
+  }
+}
+
+@media (min-width: 1200px) {
+  .login,
+  .twitter,
+  .google,
+  .facebook,
+  .custom-text-input-container,
+  .separator {
+    width: 22%;
+  }
+
+  .login-mull-logo {
+    width: 10%;
+  }
+}
```

Comment:
Please remove this code and try implementing the `.page-container-no-nav` suggestion I gave earlier.

I know that it might look a bit wide on desktop, but we can fix that by adding more breakpoints in another pr. For now just stick with what was done in the create event page